docs/develop/excel-async-model.md

```diff
@@ -11,11 +11,11 @@ localization_priority: Normal
 This article will teach you how to write scripts using the legacy, async, APIs. These APIs have the same core functionality as the standard, synchronous Office Scripts APIs, but they require that your script control the data synchronization between the script and the workbook.
 
 > [!IMPORTANT]
-> The async model is significantly more complicated than the standard Office Scripts APIs. Unless you're working with code that uses this older model, we highly recommend using the [standard API model](scripting-fundamentals.md) instead of the Office Scripts Async APIs.
+> The async model can only be used with scripts created before the implementation of the current [API model](scripting-fundamentals.md?view=office-scripts). Scripts are permanently locked to the API model they have upon creation. This also means that if you want to convert a legacy script to the new model, you must use a brand new script.
```

Comment:
Might still be worth mentioning async is harder - or making a recommendation to transition async to sync if that would be helpful to people? (Not sure)

Reply:
Yeah, good call.